src/Symfony/Bundle/FrameworkBundle/Console/Descriptor/TextDescriptor.php

```diff
@@ -38,47 +38,78 @@
  */
 class TextDescriptor extends Descriptor
 {
+    private const VERB_COLORS = [
+        'ANY' => 'red',
```

Comment:
According to your screenshot 'ANY' is still white, IMO it's fine to keep it in a neutral color like that